api/kyc/api.go

```diff
@@ -164,6 +159,12 @@ func (u *UserRepresentation) ImportFromKeycloak(ctx context.Context, kcUser *kc.
 	var accreditations = u.Accreditations
 	var locale = u.Locale
 	var businessID = u.BusinessID
+	var birthLocation = u.BirthLocation
```

Comment:
Readability is more and more heavy... May be we could write something like:
var birthLocation = defaultIfNil(kcUser.GetAttributeString(constants.AttrbBirthLocation), u.BirthLocation)

It would divide number of lines of code per 4. (and reduce number of branches)

Reply:
Good idea, this method is indeed too long

Reply:
done